airbyte-cdk/python/unit_tests/singer/test_singer_source.py

```diff
@@ -95,7 +95,7 @@ def test_singer_discover_metadata(mock_read_catalog):
     _user_stream = airbyte_catalog.streams[0]
     _roles_stream = airbyte_catalog.streams[1]
 
-    assert _user_stream.supported_sync_modes is None
+    # assert _user_stream.supported_sync_modes is None
```

Comment:
should this assert the right modes are supported?

Reply:
I'm not sure why this assertion was declared. There is probably an edge case with singer and supported sync modes that I'm unaware of.